pliers/extractors/text.py

```diff
@@ -402,3 +402,33 @@ def _extract(self, stim):
 
         return ExtractorResult(features_list, stim, self,
                                features=self.features, orders=order_list)
+
+class WordCounterExtractor(TextExtractor):
+
+    ''' Extracts number of times word has occurred within text
```

Comment:
Maybe clarify that it's the number of times each unique word occurs–current wording is easy to interpret as matching a specified word.